packages/components/tooltip/README.md

```diff
@@ -180,75 +180,51 @@ export {
+| `isOpen`               | `boolean`                                                                                                                                                                         |          |           |                                                                                                                                                                                                          |
+| `placement`            | `union`<br/>Possible values:<br/>`, 'top', 'top-start', 'top-end', 'right', 'right-start', 'right-end', 'bottom', 'bottom-start', 'bottom-end', 'left', 'left-start', 'left-end'` |          | `'top'`   | How the tooltip is positioned relative to the child element.                                                                                                                                             |
+| `title`                | `string`                                                                                                                                                                          |    ✅    |           | The message to show in the tooltip.                                                                                                                                                                      |
+| `modifiers`            | `Modifiers`                                                                                                                                                                       |          |           | Provides a way to fine-tune an appearance of underlying Popper tooltip element. For more information, please check [Popper.js documentation](https://popper.js.org/popper-documentation.html#modifiers). |
+| `components`           | `Object`<br/>[See signature.](#signature-components)                                                                                                                              |          |           | Customize the appearance of certain elements of the tooltip.                                                                                                                                             |
+| `horizontalConstraint` | `union`<br/>Possible values:<br/>`, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 'scale', 'auto'`                                                                       |          | `'scale'` | Horizontal size limit of the tooltip.                                                                                                                                                                    |
```

Comment:
```suggestion
| `horizontalConstraint` | `union`<br/>Possible values:<br/>` 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 'scale', 'auto'`                                                                       |          | `'scale'` | Horizontal size limit of the tooltip.                                                                                                                                                                    |
```

Reply:
This is a generator quirk. will not fix here.